apps/auth/src/pages/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { signIn, signOut, useSession } from 'next-auth/react'
+
+import { Button, H1 } from '@uzh-bf/design-system'
+import Image from 'next/image'
+import { useRouter } from 'next/router'
+import Footer from 'shared-components/src/Footer'
+
+function SignInOutButton() {
+  const router = useRouter()
+
+  const { data: session } = useSession()
+
+  if (session) {
+    return (
+      <>
+        Signed in as {session?.user?.email} <br />{' '}
+        <Button onClick={() => signOut()}>Sign out</Button>
+      </>
+    )
+  }
+
+  return (
+    <Button
+      className={{ root: 'p-4' }}
+      onClick={() =>
+        signIn(process.env.NEXT_PUBLIC_EDUID_ID, {
+          callbackUrl:
+            (router.query?.redirectTo as string) ??
```

Comment:
allow to pass a redirectTo path in the URL -> this will need to be escaped when we want to use it (URL encoding), but this is just for first testing